docling_ibm_models/layoutmodel/layout_predictor.py

```diff
@@ -75,6 +76,19 @@ def __init__(
         self._threshold = 0.6  # Score threshold
         self._image_size = 640
         self._size = np.asarray([[self._image_size, self._image_size]], dtype=np.int64)
+        self._use_cpu_only = use_cpu_only or ("USE_CPU_ONLY" in os.environ)
+
+        # Decide the execution providers
+        if not self._use_cpu_only:
+            providers = ["CUDAExecutionProvider", "CPUExecutionProvider"]
```

Comment:
this still looks like onnx specific content

Reply:
removed these, please update